### PR TITLE
style(console): adjust welcome page style

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

adjust welcome page style
1. replace the background img
2. set the background img size contain
3. update the position of the content block

image

## Testing

test locally
@logto-io/eng

### CODE REVIEW

File: packages/console/src/pages/Welcome/index.module.scss [hunk 1]
```diff
@@ -6,32 +6,28 @@
...
   flex-direction: column;
   min-width: 1280px;
+  min-height: 806px;
 
   &.light {
     background-image: url('../../assets/icons/welcome.svg');
-    background-position: right 0 bottom 40px;
+    background-position: right bottom;
+    background-size: contain;
     background-repeat: no-repeat;
```
Comment (gao-sun) [inline packages/console/src/pages/Welcome/index.module.scss:15]: nit: can use a shared wrapper for these three props?

File: packages/console/src/pages/Welcome/index.module.scss [hunk 2]
```diff
@@ -43,12 +39,20 @@
...
   flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.placeHolderTop {
+  flex: 1;
+}
+
+.placeHolderBottom {
```
Comment (gao-sun) [inline packages/console/src/pages/Welcome/index.module.scss:50]: use placeholder as one word should be ok

**Review by gao-sun — APPROVED** [review]
code looks good to me, confirm with design before merging

File: packages/console/src/pages/Welcome/index.module.scss (hunk 1 above)
Comment (simeng-li) [reply packages/console/src/pages/Welcome/index.module.scss]: updated

File: packages/console/src/pages/Welcome/index.module.scss (hunk 2 above)
Comment (simeng-li) [reply packages/console/src/pages/Welcome/index.module.scss]: updated

**Pull request merged**